### DISCUSSION TITLE
How do I disable the dimming of not-focused windows?

### DISCUSSION THREAD

**Discussion by JohannesPertl** [regolith-linux/regolith-desktop#689] [Q&A] [answered]
Whenever I focus another window, the other ones get darker. How do I disable this?

**Comment by Tumetsu** [comment, marked as answer]
That is compositor setting. You can read about compositors here: https://regolith-desktop.com/docs/howtos/customize-compositor/
I'm using Picom personally so I edited the config file in `/home/<home>/.config/regolith2/picom/config`. There is "Opacity" section which I edited to following:
```
#################################
#
# Opacity
#
#################################

inactive-opacity = 1;
active-opacity = 1;
frame-opacity = 1;
inactive-opacity-override = false;

# Dim inactive windows. (0.0 - 1.0)
inactive-dim = 0.00;
# Do not let dimness adjust based on window opacity.
inactive-dim-fixed = true;
```
